src/Selenium2Library/keywords/element.py

```diff
@@ -217,6 +217,23 @@ def element_should_be_enabled(self, locator):
             raise AssertionError("Element '%s' is disabled." % (locator))
 
     @keyword
+    def element_should_be_focused(self, locator):
+        """Verifies that element identified with `locator` is focused.
+
+        Key attributes for arbitrary elements are `id` and `name`. See
+        `introduction` for details about locating elements.
+
+        New in SeleniumLibrary 2.0.1.
+        """
+        element = self.find_element(locator)
+        try:
+            focused = self.browser.execute_script('return document.activeElement;')
+        except (IOError, RuntimeError) as err:
+            raise AssertionError("Internal Error: '%s'" % (err))
```

Comment:
Lines 229 - 232 could be replaced with this: https://github.com/SeleniumHQ/selenium/blob/fb6c35ed4974adc2c2f8308ee9d78fc4f6f3004a/py/selenium/webdriver/remote/switch_to.py#L34